main.go

```diff
@@ -37,7 +37,22 @@ func main() {
 	fmt.Printf("Required approvers: %s\n", requiredApproversRaw)
 	approvers := strings.Split(requiredApproversRaw, ",")
 
-	apprv, err := newApprovalEnvironment(client, repoFullName, repoOwner, runID, approvers)
+	minimumApprovalsRaw := os.Getenv(envVarMinimumApprovals)
+	minimumApprovals := len(approvers)
+	if minimumApprovalsRaw != "" {
+		minimumApprovals, err = strconv.Atoi(minimumApprovalsRaw)
+		if err != nil {
+			fmt.Printf("error parsing minimum number of approvals: %v\n", err)
+			os.Exit(1)
+		}
+	}
+
+	if minimumApprovals > len(approvers) {
+		fmt.Printf("warning: minimum required approvals (%v) is greater than the total number of approvers (%v)\n", minimumApprovals, len(approvers))
```

Comment:
I agree with the exiting with a non-zero status code but this message shouldn't be "warning", it should be "error".

```suggestion
		fmt.Printf("error: minimum required approvals (%v) is greater than the total number of approvers (%v)\n", minimumApprovals, len(approvers))
```

Reply:
Agreed, I have updated this to an error